pandas/io/tests/sas/test_sas7bdat.py

```diff
@@ -65,6 +65,33 @@ def test_from_iterator(self):
                 df = rdr.read(3)
                 tm.assert_frame_equal(df, df0.iloc[2:5, :])
 
+    def test_iterator_loop(self):
+        # github #13654
+        for j in 0, 1:
+            for k in self.test_ix[j]:
+                for chunksize in 3, 5, 10, 11:
+                    fname = os.path.join(self.dirpath, "test%d.sas7bdat" % k)
+                    with open(fname, 'rb') as f:
+                        byts = f.read()
+                    buf = io.BytesIO(byts)
```

Comment:
Is there a reason you read the file into a bytes object and not just pass the `fname` to ``read_sas`` ?